code/modules/mob/living/simple_animal/friendly/slime.dm

```diff
@@ -23,11 +23,8 @@
 
 /mob/living/simple_animal/slime/attackby(var/obj/item/weapon/slimeparapotion/O as obj, var/mob/user as mob)
 	if(istype(O))
-		var/obj/item/weapon/paraslime/F = new /obj/item/weapon/paraslime(get_turf(src))
-		F.icon_state = icon_dead
-		F.name = "paralyzed [colour] slime"
-		F.stored = src
-		forceMove(F)
+		canmove = 0
+		icon_state = "[src.colour] baby slime dead"
```

Comment:
Could do with some feedback to the user to tell them what's happened, like 'the [src] appears to shrivel up and stiffen into something more handleable.'